heroku/cli
# refactor(pg-v5): Move command `pg:credentials:url` to oclif

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001bD6OlYAK/view

This converts `pg:credentials:url` to oclif and updates its associated tests.

### Testing

1) Pull down this branch
2) Run `./bin/run pg:credentials:url -a APP` with an app that has a pg db 
3) Verify the command succeeds, displaying info about the db's creds
4) Compare with prod
